### PR TITLE
update axios version

### PR DESCRIPTION
**Pull request by rrez2002**

### CODE REVIEW

**Review by thomasSokolowski — APPROVED** [review]

**Review by juan-turk-simplisafe — COMMENTED** [review]
Can you please eliminate the spaces (indentation) so it is easier to see what the changes are?

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "xml-crypto": "^3.0.0"
   },
   "peerDependencies": {
-    "axios": "^0.27.2"
+    "axios": ">=1.5.0"
```
Comment (alfaproject) [inline package.json:22]: Still not sure why this is a peer dependency. This module CAN NOT function without axios, it clearly is a dependency, lol

Comment (joanklikin) [reply package.json:22]: https://github.com/vpulim/node-soap/pull/1179

**Review by juan-turk-simplisafe — APPROVED** [review]
Any news about this? can we merge this

**Comment by boukmi** [comment]
Any update? can this be merged?
Thank you

**Comment by Dragonox77** [comment]
Bump

**Comment by boxexchanger** [comment]
@jsdevel +1

**Comment by swiesmann** [comment]
We would also appreciate the PR merge 👍 as there is a known vulnerability for the axios version that is in use. 

https://github.com/advisories/GHSA-wf5p-g6vw-rhxx

**Comment by Caryyon** [comment]
This dependency vulnerability is a show stopper for our app, without being able to update axios our app won't pass it's pipeline vulnerability scans.

**Comment by invariants** [comment]
This PR with essentialy the same goal is open for almost one year: https://github.com/vpulim/node-soap/pull/1212 (has also been kept up to date). I think this project is dead and needs to be forked.

**Comment by juan-turk-simplisafe** [comment]
Is there already a fork of this repo that is updated?
This repo has a vulnerable version of Axios and I need to update it in my project as soon as possible.

**Comment by lalitsharma309** [comment]
Any update? can this be merged?
Thank you

**Review by boxexchanger — APPROVED** [review]

**Comment by jackhollowaypls** [comment]
Please can this get approved. It's the only package left in my project that has a dependancy on this old version of axios, all other packages have updated

**Review by lalitsharma309 — APPROVED** [review]

**Comment by Dragonox77** [comment]
Can you release it ?

**Comment by mfuqua3** [comment]
Bump. Can this please get merged and released.

**Comment by Deprasos** [comment]
Same here still waiting for this to be merged and released 👍

File: package.json (hunk 1 above)
Comment (simhnna) [reply package.json:22]: That point is no longer valid. this package can easily state that it directly depends on axios >= 1.5.0

In my project I would be able to use any axios version that satisfies the constraint

**Comment by SiRocke** [comment]
Bump!

**Comment by alexHerrmio** [comment]
Please merge it ❤️

**Comment by juan-turk-simplisafe** [comment]
Bump!

**Comment by dkbhadeshiya** [comment]
This really needs to be merged since older axios versions are having vulnerabilities.

**Comment by Dragonox77** [comment]
Guys, it is probably better to use another lib, since the author do not seems to be active and probably do not care.

If a vulnerability is discovered tomorrow, we can't wait for the author to merge the fix in 3 months. This lib should be marked as deprecated or inactive. Maybe we should ask NPM to pressure them if they want to keep their package active.

**Comment by hamidouikene** [comment]
@Dragonox77, could you suggest any other lib that we can use here?

**Comment by Dragonox77** [comment]
@hamidouikene 
> @Dragonox77, could you suggest any other lib that we can use here?

https://www.npmjs.com/package/strong-soap (updated 7 days ago)
https://www.npmjs.com/package/easy-soap-request (updated 1 month ago)

**Comment by juan-turk-simplisafe** [comment]
I agree with @Dragonox77, we are moving to another library.

**Comment by swiesmann** [comment]
Thank you for merging, @vpulim ! 👍 Looking forward to the release.

**Comment by JackHammer29** [comment]
@vpulim Any update on the release please?

**Comment by w666** [comment]
Hi All, I will be helping to maintain this project for some time, discussing details with @vpulim atm. Hopefully I will release this soon.

**Pull request merged**